VideoAPI/VideoApi.Services/ConsultationService.cs

```diff
@@ -0,0 +1,44 @@
+using Microsoft.Extensions.Logging;
+using System;
+using VideoApi.Domain.Enums;
+using VideoApi.Services.Contracts;
+using VideoApi.Services.Kinly;
+using Task = System.Threading.Tasks.Task;
+
+namespace VideoApi.Services
+{
+    public class ConsultationService : IConsultationService
+    {
+        private readonly IKinlyApiClient _kinlyApiClient;
+        private readonly ILogger<ConsultationService> _logger;
+
+        public ConsultationService(IKinlyApiClient kinlyApiClient, ILogger<ConsultationService> logger)
+        {
+            _kinlyApiClient = kinlyApiClient;
+            _logger = logger;
+        }
+
+
+        public Task TransferParticipantAsync(Guid conferenceId, Guid participantId, VirtualCourtRoomType fromRoom,
+            VirtualCourtRoomType toRoom)
+        {
+            _logger.LogInformation(
+                "Transferring participant {participantId} from {fromRoom} to {toRoom} in conference: {conferenceId}",
+                participantId, fromRoom, toRoom, conferenceId);
+
+            var request = new TransferParticipantParams
+            {
+                From = fromRoom.ToString(),
```

Comment:
The room label must be provided, not the room type when transfering

Reply:
There is only one WR so there is no concept of a room label for those. Think room type is sufficient there. 
I can't think of a scenario where a leave will take you to any other room than WR

Reply:
updated to string